src/tools/wasp_sqlite.cc

```diff
@@ -80,11 +82,11 @@ struct Tool {
 
   size_t file_offset(SpanU8 data);
 
-  std::string filename;
+  std::vector<string_view> filenames;
   Options options;
-  SpanU8 data;
   ErrorsNop errors;  // XXX
-  LazyModule module;
+  std::vector<LazyModule> modules;
```

Comment:
Do you need to keep the modules around?

Reply:
No idea! Seems no, we only ever kept the initial `module` around to close over it, which is handled by converting it to a pointer.